cli/src/run_testenv.rs

```diff
@@ -278,111 +277,70 @@ pub async fn initialize_test_organization(
     Ok([alice_device, other_alice_device, bob_device])
 }
 
+pub enum TestenvConfig {
+    ServerAlreadyRunning(String),
```

Comment:
I'd call it something like 
```suggestion
    ConnectToServer(String),
```